### PR TITLE
Update AboutUs Page. Fix indentation bug. Fix error messages

### LINKED ISSUE #159: [PE-D][Tester D] Error messages should be plural in the list commands
-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: abhityakrishnaraj/ped#5

### PR DESCRIPTION
**Pull request by zoranabc201**

### CODE REVIEW

File: src/main/java/seedu/duke/Ui.java [hunk 1]
```diff
@@ -234,7 +234,7 @@ public void displayOnePropertyInPairShort(Property property) {
 
     public void displayOneClientName(Client client, int i) {
         System.out.print(i + FULL_STOP);
-        System.out.println(client.getClientName());
+        System.out.println(SPACE + client.getClientName());
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/Ui.java:237]: Good use of constant here.

**Review by ngdeqi — COMMENTED** [review]
LGTM

File: src/main/java/seedu/duke/Ui.java (hunk 1 above)
Comment (OVReader) [reply src/main/java/seedu/duke/Ui.java:237]: Agreed.

File: docs/AboutUs.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
 # About us
 
-Display |       Name        | Github Profile | Portfolio 
---------|:-----------------:|:--------------:|:---------:
-![](https://via.placeholder.com/100.png?text=Photo) | Wilson Ng Jing An | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) |  Syed Omar Zoran  | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Ang Jia Le Marcus | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) |     Ng De Qi      | [Github](https://github.com/ngdeqi) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Felicia Beatrice Budiawan | [Github](https://github.com/FeliciaBeatrice) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) |  Syed Omar Zoran  | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
```
Comment (OVReader) [inline docs/AboutUs.md:10]: Good fix on duplicated entries.

**Review by OVReader — APPROVED** [review]
Overall LGTM

**Pull request merged**